eng/pipelines/publish.yml

```diff
@@ -0,0 +1,70 @@
+trigger:
+  branches:
+    include:
+    - feature/*
+
+variables:
+  BlobFeedUrl: https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json
```

Comment:
Why is this needed? This is an old feed which is no longer used.

Reply:
Doesn't look like this variable is used any longer in our template https://dev.azure.com/azure-sdk/internal/_git/azure-sdk-build-tools?path=/pipelines/stages/net-release-to-feed.yml so it should be safe to remove here and in the build.yml file as well.

Reply:
Will delete it.